L1Trigger/L1TGlobal/src/CorrCondition.cc

```diff
@@ -646,13 +659,15 @@ const bool l1t::CorrCondition::evaluateCondition(const int bxEval) const {
           phiIndex1 = (candMuVec->at(cond1bx, obj1Index))->hwPhiAtVtx();  //(*candMuVec)[obj0Index]->phiIndex();
           etaIndex1 = (candMuVec->at(cond1bx, obj1Index))->hwEtaAtVtx();
           etIndex1 = (candMuVec->at(cond1bx, obj1Index))->hwPt();
+	  uptIndex1 = (candMuVec->at(cond1bx, obj1Index))->hwPtUnconstrained(); // Added by R.Cavanaugh for displaced muons
```

Comment:
Please remove meaningless comments  for understanding the code.  Git versioning system keeps track of authors of commits.

Reply:
@rekovic 
Many thanks for the helpful request to remove comments.  Done.